src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -129,8 +129,11 @@ export class BrowsingContextProcessor {
       // debugger  and ignore them.
       targetCdpClient
         .sendCommand('Runtime.runIfWaitingForDebugger')
```

Comment:
let's change to Promise.all([runIfWaiting, detach])? I am not sure that needs to be sequential

Reply:
Not sure either. Puppeteer does this in sequence.